modules/virt-configuring-cluster-dpdk.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-cluster-dpdk_{context}"]
+= Configuring a cluster for DPDK workloads
+
+Use this procedure to configure an {product-title} cluster to run Data Plane Development Kit (DPDK) workloads.
+
+:FeatureName: Configuring a cluster for DPDK workloads
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the SR-IOV Network Operator.
+* You have installed the Node Tuning Operator.
+* The compute nodes have pre-allocated huge pages and Network Interface Cards (NICs) compatible with SR-IOV.
+
+.Procedure
+. Create an `SriovNetworkNodePolicy` object with the `spec.deviceType` field set to `vfio-pci`.
+. Create a `RuntimeClass` resource by using the `PerformanceProfile` object of the Node Tuning Operator.
+. Set the container runtime configuration for the `virt-launcher` pod by adding the following annotation to the `HyperConverged` custom resource (CR):
```

Comment:
```suggestion
. Map your compute nodes topology - understand to which NUMA node your SR-IOV NIC is connected to.
. Label a subset of your worker nodes as DPDK-enabled, e.g. `oc label nodes <node_name> node-role.kubernetes.io/worker-dpdk=""`.
. Create a new `MachineConfigPool` applied to the labeled nodes.
. Create a new `PerformanceProfile` applied to the labeled nodes and the previously configured machine config pool. This profile should select `isolated` CPUs for your DPDK application and `reserved` CPUs for house keeping. Refer to the `PerformanceProfile` documentation for more detail.  
. Create an `SriovNetworkNodePolicy` object with the `spec.deviceType` field set to `vfio-pci`.
. Set the container runtime configuration for the `virt-launcher` pod by adding the following annotation to the `HyperConverged` custom resource (CR):
```

XXX: This may have support implications. We are testing an alternative option. If that option will look ok, we will just need to drop this last step and add a note to the "Create a new `PerformanceProfile`"